src/test/java/org/fcrepo/importexport/importer/ImporterTest.java

```diff
@@ -144,8 +165,19 @@ public void testImportBinary() throws Exception {
     }
 
     @Test
+    public void testImportExternalResource() throws Exception {
+        final Importer importer = new Importer(externalResourceArgs, clientBuilder);
+        importer.run();
+        verify(client).put(externalResourceURI);
+        verify(externalResourceBuilder).body(eq(new File(externalFilesDir, "rest/ext1.external")),
+                                eq("message/external-body"));
+        verify(client).put(externalResourceDescriptionURI);
+    }
+
+    @Test
     public void testImportWithoutBinaries() throws Exception {
-        final Importer importer = new Importer(noBinaryArgs, clientBuilder);
+        binaryArgs.setIncludeBinaries(false);
```

Comment:
This looks like a good way to avoid a bunch of redundant objects above — but are you sure the tests are run in a predictable order and this won't interfere with the other tests that expect it to include binaries?